test/lib/redis-store-spec.js

```diff
@@ -257,6 +257,30 @@ describe('del', function () {
     });
   });
 
+  it('should delete multiple values for a given array of keys', function (done) {
+    redisCache.set('foo', 'bar', function () {
+      redisCache.set('bar', 'baz', function () {
+        redisCache.set('baz', 'foo', function () {
+          redisCache.del(['foo', 'bar', 'baz'], function (err) {
+            assert.equal(err, null);
+            done();
```

Comment:
## Suggestion:
Maybe we should fetch the deleted keys to check if they were removed or not.